core/src/test/scala/org/apache/spark/SparkContextSuite.scala

```diff
@@ -366,6 +366,29 @@ class SparkContextSuite extends SparkFunSuite with LocalSparkContext with Eventu
     }
   }
 
+  test("SPARK-33084: add jar when path contains comma") {
+    withTempDir { tmpDir =>
+      val tmpJar = File.createTempFile("Test,UDTF", ".jar", tmpDir)
+      sc = new SparkContext(new SparkConf().setAppName("test").setMaster("local"))
+      sc.addJar(tmpJar.getAbsolutePath)
+      assert(sc.listJars().exists(_.contains("Test,UDTF")))
```

Comment:
https://github.com/apache/spark/pull/29966#issuecomment-739631644 hm, I think it would be better to check jars are added correctly in executors, too.

Reply:
> [#29966 (comment)](https://github.com/apache/spark/pull/29966#issuecomment-739631644) hm, I think it would be better to check jars are added correctly in executors, too.

Yea， will update a test case later

Reply:
> [#29966 (comment)](https://github.com/apache/spark/pull/29966#issuecomment-739631644) hm, I think it would be better to check jars are added correctly in executors, too.

How about UT https://github.com/apache/spark/pull/29966/commits/57c351d076da51fd8c41323f16ff8d8fbc4b964b?